Add SD-JWT selective disclosure guidance for credential privacy (#40)
w3c-ccg/did-pkh: did-pkh-method-draft.md

### Key Derivation Lacks Proof

> Some implementations might utlize a key derivation function, e.g., when

Reviewer: ```suggestion
Some implementations might use a key derivation function, e.g., when
```

converting from an ed25519 public key to a Curve25519 ECDH key, used in the
keyAgreement verification method (in did-key, and perhaps in future CAIP-based

> pkh implementations). It is expected that this is a relatively safe operation,
> but implementers might consider that there exists no mathematical proof that
> confirms this assumption.

Reviewer: ```suggestion
pkh implementations). This operation is expected to be relatively safe,
but implementers might consider that there exists no mathematical proof that
confirms this expectation.
```
